Bump gz-cmake and others in jetty

scpeters opened the pull request:
Part of https://github.com/gazebo-tooling/release-tools/issues/1309.

scpeters commented:
my naive change in https://github.com/gazebosim/gz-gui/pull/683/commits/8447769f899a8573f82dd34d17837e16a42c53e2 did not fix the colcon build since I believe the qt6 packages are not yet in rosdep

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ci__colcon_any-manual_ubuntu_noble_amd64&build=29)](https://build.osrfoundation.org/job/ci__colcon_any-manual_ubuntu_noble_amd64/29/) https://build.osrfoundation.org/job/ci__colcon_any-manual_ubuntu_noble_amd64/29/

scpeters commented:
trying again with `<depend>` tags commented out that don't have rosdep keys and manually installing all the qt6 packages via jenkins build argument

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ci__colcon_any-manual_ubuntu_noble_amd64&build=30)](https://build.osrfoundation.org/job/ci__colcon_any-manual_ubuntu_noble_amd64/30/) https://build.osrfoundation.org/job/ci__colcon_any-manual_ubuntu_noble_amd64/30/

j-rivero commented:
> trying again with `<depend>` tags commented out that don't have rosdep keys and manually installing all the qt6 packages via jenkins build argument
> 
>     * [![Build Status](https://camo.githubusercontent.com/e1468156ceac87cdb8a9639664b0f4e5d73bf1eb4f59ba8f65fc6a1af40c00f1/68747470733a2f2f6275696c642e6f7372666f756e646174696f6e2e6f72672f6275696c645374617475732f69636f6e3f6a6f623d63695f5f636f6c636f6e5f616e792d6d616e75616c5f7562756e74755f6e6f626c655f616d643634266275696c643d3330)](https://build.osrfoundation.org/job/ci__colcon_any-manual_ubuntu_noble_amd64/30/) https://build.osrfoundation.org/job/ci__colcon_any-manual_ubuntu_noble_amd64/30/

I think that the gz-sensors code is broken here:

```
/tmp/ws/src/gz-sensors/src/CameraSensor.cc:719:19: error: ‘const class sdf::v16::Camera’ has no member named ‘OpticalFrameId’
```

OpticalFrameId was removed in sdf16, replaced by just the Camera Frame Id.

scpeters commented:
sorry these links are outdated

* https://github.com/gazebosim/gz-sensors/pull/525/commits/1bda368eee9828a6ebf923f2d783d0560023a658
* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ci__colcon_any-manual_ubuntu_noble_amd64&build=36)](https://build.osrfoundation.org/job/ci__colcon_any-manual_ubuntu_noble_amd64/36/) https://build.osrfoundation.org/job/ci__colcon_any-manual_ubuntu_noble_amd64/36/

scpeters commented:
I'm not sure why that failed CI build is listed; everything else is 🟢

scpeters commented:
> I think that the gz-sensors code is broken here:
> 
> ```
> /tmp/ws/src/gz-sensors/src/CameraSensor.cc:719:19: error: ‘const class sdf::v16::Camera’ has no member named ‘OpticalFrameId’
> ```
fixed in https://github.com/gazebosim/gz-sensors/commit/1bda368eee9828a6ebf923f2d783d0560023a658

> OpticalFrameId was removed in sdf16, replaced by just the Camera Frame Id.

iche033 commented:
I think we no longer need the `_test_conda_gz_gui-pr-clowin` job now that there is `gz_gui-pr-cnlwin`
